geomesa-ui/src/main/ui/README.md

```diff
@@ -5,8 +5,17 @@
 ***
 
 ## Getting started
-_(Coming soon)_
+1. npm install
+1. bower install
+1. grunt server
+1. enjoy by using default localhost 9090
 
 ## Documentation
-_(Coming soon)_
+* [Google Doc](https://docs.google.com/document/d/1o3rwxbdxb245YhkF55cPLQNTF3ushZFlivsJ3f0yIVk/)
```

Comment:
Does that mean to move the content from Google doc to README in markdown format?